requirements/base.txt

```diff
@@ -2,6 +2,7 @@
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
 numpy >1.20.0
+packaging
```

Comment:
any min version?

Reply:
Not sure, the API is pretty stable so I imagine these functions have existed for a long time. Do you test the minimum version in CI? I could try a binary search for a minimum version that passes the tests.

Reply:
```suggestion
packaging >17.1
```
so, set it at least 5 years back?

Reply:
Should we do the same for typing-extensions? I hate to choose an arbitrary version...

Reply:
let's pick one which was released 3-5 years ago :)